index.css

```diff
@@ -388,3 +403,213 @@ label {
   font-weight: 400;
   line-height: 28px;
 }
+
+.list_style {
+  list-style: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .nav1 {
+    display: none;
+  }
+
+  .nav2 {
+    display: block;
+    display: flex;
+    gap: 20px;
+  }
+
+  .nav2 a {
+    font-size: 15px;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  .section1 {
+    display: flex;
+    justify-content: center;
+  }
+
+  .section1 h1 {
+    font-size: 48px;
+    color: #172b4d;
+    line-height: 60px;
+    font-weight: 700;
+    width: 420px;
+  }
+
+  .section1 p {
+    color: #344563;
+    font-size: 13px;
+    line-height: 28px;
+    width: 580px;
+    font-weight: 400;
+    align-items: left;
+    padding: 50px 0 0 0;
+  }
+
+  .works_section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 90%;
+    margin: auto;
+    padding: 20px 0 30px 0;
+  }
```

Comment:
Please follow the works_section article separation as displayed in Figma 😃:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/199867066-a528410c-ef76-480c-9a97-e5f498a2f8bb.png)  | ![image](https://user-images.githubusercontent.com/42799579/199867185-4559b757-0d87-4445-86b3-7076d597b813.png)  | 

Notice how the original background color of the body is different, and each article is a different card with rounded borders 😉.